packages/adapter-cloudflare/ambient.d.ts

```diff
@@ -1,13 +1,14 @@
-import { CacheStorage, IncomingRequestCfProperties } from '@cloudflare/workers-types';
+import { CacheStorage, CfProperties } from '@cloudflare/workers-types';
 
 declare global {
 	namespace App {
 		export interface Platform {
+			env: unknown;
 			context: {
 				waitUntil(promise: Promise<any>): void;
 			};
 			caches: CacheStorage;
-			cf?: IncomingRequestCfProperties;
+			cf?: CfProperties;
```

Comment:
why this change? I don't think this is correct

(you also didn't change this above in the 60-adapter-cloudflare.md file: https://github.com/sveltejs/kit/pull/13634/files#diff-cf2e257a80a84245d0cdf7c44a3fb02e66db6781566df9b8670ee22ea5bd580dR122)

Reply:
You're right. I'll revert this change. Seems to just be a type issue we can ignore.